docs/pages/careers.tsx

```diff
@@ -147,16 +147,57 @@ const openRolesData = [
     title: 'Engineering',
     roles: [
       {
-        title: 'React Engineer - X',
+        title: 'React Tech Lead - Core',
         description:
-          'You will strengthen the advanced components team, build new ambitious complex features, work on strategic problems, and help grow the adoption.',
-        url: '/careers/react-engineer-x/',
+          'You will lead the development of MUI Core, positioning the library as the industry standard for design teams while doubling the adoption of the library.',
+        url: '/careers/react-tech-lead-core/',
+      },
+      {
+        title: 'React Engineer - Core',
+        description:
+          'You will strengthen the core components team, e.g. collaborate with the community to land contributions.',
+        url: '/careers/react-engineer-core/',
+      },
+      {
+        title: 'React Tech Lead - xGrid',
+        description:
+          'You will lead the development of the data grid, positioning the component as the next industry standard.',
+        url: '/careers/react-tech-lead-x-grid/',
+      },
+      {
+        title: 'React Engineer - xGrid',
+        description:
+          'You will strengthen the data grid team, build new ambitious complex features, work on strategic problems, and help grow the adoption.',
+        url: '/careers/react-engineer-x-grid/',
       },
       {
         title: 'Product Engineer - Store',
-        description: 'You will lead the technical and operational development of MUI Store.',
+        description:
+          'You will lead the technical, product, and operational development of the store.',
         url: '/careers/product-engineer/',
       },
+      {
+        title: 'Accessibility Engineer',
+        description:
+          'You will become our go-to-expert for accessibility. We need to ensure we pass WCAG 2.1 level AA across products.',
+        url: '/careers/accessibility-engineer/',
+      },
+    ],
+  },
+  {
+    title: 'Design',
+    roles: [
+      {
+        title: 'Lead Designer',
+        description:
+          'We are looking for a lead designer to own most of our design work and manage a small design team.',
```

Comment:
To match the "you will" phrasing of the other descriptions, and to avoid repeating "design":

```suggestion
          'You will own most of MUI's design work and manage a small team of designers.',
```